dashboard/app/views/devise/registrations/new.html.haml

```diff
@@ -120,7 +120,7 @@
     %br/
     %br/
 
-%script{src: minifiable_asset_path('js/signup.js')}
+%script{src: minifiable_asset_path('js/devise/registrations/new.js')}
```

Comment:
some of these paths are more edge casey things that i'm not sure that we have test coverage for. might be worth trying to manually validate some/most of these
(is this one just signin?)